copter/source/docs/gcs-failsafe.rst

```diff
@@ -3,8 +3,9 @@
 ============
 GCS Failsafe
 ============
+Significant improvements have been made in Copter-4.0 and later for Ground Control Failsafe .
 
-The Ground Station Control (GCS) failsafe controls how Copter will behave if contact with the GCS is lost.  The GCS failsafe monitors the time since the last mavlink heartbeat from the GCS.  If no heartbeat is received in over 2.5 seconds, the GCS failsafe event will trigger based on your parameter settings. Note that if no GCS is ever connected, the GCS failsafe will remain inactive regardless of parameter settings.
+The Ground Station Control (GCS) failsafe controls how Copter will behave if contact with the GCS is lost.  The GCS failsafe monitors the time since the last mavlink heartbeat from the GCS.  If no heartbeat is received in over 5 seconds or 2.0 seconds if the vehicle is being controlled by Joystick thru the GCS, the GCS failsafe event will trigger based on your parameter settings. Note that if no GCS is ever connected, the GCS failsafe will remain inactive regardless of parameter settings.
```

Comment:
We should remove the mention of the GCS failsafe sometimes being 2sec.. but it's actually always 5sec

Reply:
will fix on next update to PR

Reply:
done